vstools install doc update (#1607)
Samsung/tizen-docs: docs/application/vstools/install.md

- Visual Studio 2017 to use Tizen 4.0 and 5.0
- Visual Studio 2019 to use Tizen 4.0 and higher
- Visual Studio 2022 to use Tizen 6.5 and higher

> - Latest Tizen Tools updates support Tizen Native and Web app creation and is provided with Tizen Studio version 4.5 and above. Make sure the same is installed or [updated](https://docs.tizen.org/application/tizen-studio/setup/update-sdk/) through Tizen package manager.

Reviewer: Also, please use relative path to the pages in tizen-docs repository.

Reviewer 2: @mijin85cho, I believe this should be the correct path.
```suggestion
- Latest Tizen Tools updates support Tizen Native and Web app creation and is provided with Tizen Studio version 4.5 and above. Make sure the same is installed or [updated](../tizen-studio/setup/update-sdk.md) through Tizen package manager.
```

Author: Thank you @omar-srbd, I have applied this modification [3de2929](https://github.com/Samsung/tizen-docs/pull/1607/commits/3de2929a507705919aaac7f8993d1128f0c97ae3).

Reviewer: @kusin10 Thanks for the update!

@omar-srbd Correct. As you can see in see in https://github.com/Samsung/tizen-docs/blob/3de2929a507705919aaac7f8993d1128f0c97ae3/docs/application/vstools/install.md, 
when you click the link, it goes to https://github.com/Samsung/tizen-docs/blob/3de2929a507705919aaac7f8993d1128f0c97ae3/docs/application/tizen-studio/setup/update-sdk.md now.

> Tizen Baseline SDK. Make sure you download and install the EXACT version.

Author: Applied suggested modification [3de2929](https://github.com/Samsung/tizen-docs/pull/1607/commits/3de2929a507705919aaac7f8993d1128f0c97ae3).

  - [Oracle Java Development Kit(JDK) 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).

  - OpenJDK 12 and OpenJFX: [OpenJDK 12 and OpenJFX Installation Guide](../tizen-studio/setup/openjdk.md#install-openjdk-for-windows).

> ### Emulator Requirements

Author: Modification applied [3de2929](https://github.com/Samsung/tizen-docs/pull/1607/commits/3de2929a507705919aaac7f8993d1128f0c97ae3)

<a name="install"></a>
## Visual Studio Tools for Tizen installation

To use the Tizen SDK tools, you must install the Visual Studio Tools for Tizen extension and the Baseline SDK.

> [!NOTE] 

> > If Tizen Studio on already installed in your computer, simply [set the Tizen Baseline SDK path](#set-tizen-baseline-sdk) to the existing SDK instead of reinstalling it.

Reviewer 2: "is" is the proper preposition here.
```suggestion
> If Tizen Studio is already installed in your computer, simply [set the Tizen Baseline SDK path](#set-tizen-baseline-sdk) to the existing SDK instead of reinstalling it.
```

> ### Installing the Extension

Reviewer 2: Please use sentence style capitalization for second heading.
```suggestion
### Installing the extension
```
